modules/javafx.graphics/src/main/java/com/sun/javafx/application/HostServicesDelegate.java

```diff
@@ -135,11 +135,16 @@ public void showDocument(final String uri) {
             String osName = System.getProperty("os.name");
             try {
                 if (osName.startsWith("Mac OS")) {
-                    Runtime.getRuntime().exec(
-                            "open " + uri);
+                    Runtime.getRuntime().exec(new String[] {
```

Comment:
not a review, but a general question:
do we have (do we want) an enum identifying the platform?

Reply:
Don't think we have an enum (maybe we should), but `PlatformUtil` already takes care of identifying the OS, also using `System.getProperty("os.name")`. You can use `PlatformUtil.isMac()`. If we make an enum, that's where is should be.

Reply:
There was a recent PR
https://github.com/openjdk/jdk/pull/13357
adding an enum deep in the jdk bowels

we probably can't use that

Reply:
No, we can't use that. More to the point, it is unrelated to this PR (and thus out of scope).

Reply:
But I do think it would be a good idea to use `PlatformUtil` here, so maybe file a cleanup issue to look for places where we explicitly parse `os.name`, and consider calling the appropriate `PlatformUtil` method instead?

Reply:
+1 for PlatformUtil.